src/hypofuzz/cov.py

```diff
@@ -56,74 +61,74 @@ def get_possible_branches(cov: coverage.CoverageData, fname: str) -> frozenset[A
         return _POSSIBLE_ARCS[fname]
 
 
-class CollectionContext:
-    """Collect coverage data as a context manager.
+is_hypothesis_file = belongs_to(hypothesis)
+is_hypofuzz_file = belongs_to(hypofuzz)
 
-    The context manager can be reused; each use updates the ``.branches``
-    attribute which will be reset on next use.
 
-    TODO: excluding Hypothesis (and fuzz) files from tracing as well
-            as results would be a small performance upgrade.
-    """
```

Comment:
I assume this is leftover from a coveragepy integration - hope you agree that rolling our own coverage collection instead is the way to go?

Reply:
yes and yes!